src/main.zig

```diff
@@ -22,12 +22,10 @@ const Allocator = std.mem.Allocator;
 
 const log = @import("log.zig");
 const App = @import("app.zig").App;
-const Http = @import("http/Http.zig");
 const Server = @import("server.zig").Server;
 const Browser = @import("browser/browser.zig").Browser;
 
 const build_config = @import("build_config");
-const parser = @import("browser/netsurf.zig");
```

Comment:
Do you use something auto to cleanup imports? or it's manual?

Reply:
I used https://github.com/DonIsaac/zlint   I was hoping it would help me find error return types that didn't need to be, but it doesn't detect that I guess. And I couldn't help myself but also remove these imports when I saw it.